CHANGELOG.md

```diff
@@ -13,8 +13,8 @@
   - This allows drop-in use of colors from most applications.
   - New methods `Color::rgb_linear` and `Color::rgba_linear` will accept colors already in linear sRGB (the old behavior)
   - Individual color-components must now be accessed through setters and getters: `.r`, `.g`, `.b`, `.a`, `.set_r`, `.set_g`, `.set_b`, `.set_a`, and the corresponding methods with the `*_linear` suffix.
-- Despawning an entity multiple times causes a debug-level log message to be emitted instead of a panic [649] [651]
-
+- Despawning an entity multiple times causes a debug-level log message to be emitted instead of a panic [649][651]
+- Runner function has to call app.initialize() to run startup systems, before entering the loop.
```

Comment:
Haha it didn't occur to me at the time of merging the PR template that we couldn't add links to prs until after they have been created. It has been called out in the past that requiring changelog entries in prs also creates merge conflicts (which is what we're seeing here).

Maybe its better to just make changelog maintenance a separate process :)